pkgs/development/libraries/science/math/magma/generic.nix

```diff
@@ -117,6 +127,9 @@ stdenv.mkDerivation {
   ];
 
   cmakeFlags = lists.optionals cudaSupport [
+    "-DGPU_TARGET=${gpuTargetString}"
+    "-DCMAKE_CUDA_ARCHITECTURES=${cudaArchitecturesString}"
```

Comment:
Dunno. I had a bunch of "respect cudaCapabilities" commits in here already, so I threw this in too